Configuration/TypoScript/constants.txt

```diff
@@ -0,0 +1,108 @@
+	catalog.filter.tree.startid =
+	# cat=ai100basic/enable/121; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-filter.tree-levels-always
+	catalog.filter.tree.levels-always =
+	# cat=ai100basic/enable/122; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-filter.tree-levels-only
+	catalog.filter.tree.levels-only =
+
+	# cat=ai100basic/links/100; type=string; label=Content baseurl:Path to the content directory or content base URL to a server
+	common.content.baseurl = uploads/tx_aimeos
+
+	# cat=ai130page/links/100; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-list.target
+	catalog.lists.pid =
+	# cat=ai130page/links/110; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-detail.target
+	catalog.detail.pid =
+	# cat=ai130page/links/120; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-stock.target
+	catalog.stock.pid =
+	# cat=ai130page/links/130; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-count.target
+	catalog.count.pid =
+	# cat=ai130page/links/140; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-session-pinned.target
+	catalog.pinned.pid =
+	# cat=ai130page/links/150; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-suggest.target
+	catalog.suggest.pid =
+	# cat=ai130page/links/160; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:basket-standard.target
+	basket.pid =
+	# cat=ai130page/links/170; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:checkout-standard.target
+	checkout.pid =
+	# cat=ai130page/links/171; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:checkout-standard-summary-option-terms.target
+	checkout.terms.pid =
+	# cat=ai130page/links/172; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:checkout-standard-summary-option-terms-privacy.target
+	checkout.terms.privacy.pid =
+	# cat=ai130page/links/180; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:checkout-confirm.target
+	checkout.confirm.pid =
+	# cat=ai130page/links/190; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:checkout-update.target
+	checkout.update.pid =
+	# cat=ai130page/links/200; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:account-favorite.target
+	account.favorite.pid =
+	# cat=ai130page/links/210; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:account-watch.target
+	account.watch.pid =
+	# cat=ai130page/links/220; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:account-download.target
+	account.download.pid =
+
+	view.pluginNamespace = ai
```

Comment:
This is something that can't be changed at all, so we shouldn't give the integrator/editor a chance to do so by accident